src/components/home/Configuration.tsx

```diff
@@ -45,191 +45,595 @@ const CodeTabItem = ({code, language}: {code: string; language: "json" | "yaml"
 
 export default Configuration
 
-const GRAPHQL_CONFIG = `schema
-  @server(port: 8000)
-  @upstream(baseURL: "http://jsonplaceholder.typicode.com") {
+const GRAPHQL_CONFIG = `schema 
+@server @upstream(baseURL: "https://jsonplaceholder.typicode.com") {
```

Comment:
Don't change this configuration on the home page as it might overwhelm the user. Add this configuration at the end of the Getting Started Guide - http://localhost:3000/docs/


Just before deploying, the user can use the whole configuration with everthing for Json Placeholder.

Reply:
Okay